From: Stefan Dirsch <sndirsch@suse.de>
Subject: Enable CONFIG_DRM_I915 on x86_64 as well

Stefan expects this hardware to be used on x86_64 boards as well in
the future, so enable the driver there as well.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

--- a/drivers/char/agp/Kconfig.old
+++ b/drivers/char/agp/Kconfig
@@ -78,7 +78,7 @@
 
 config AGP_INTEL
 	tristate "Intel 440LX/BX/GX, I8xx and E7x05 chipset support"
-	depends on AGP && X86 && !X86_64
+	depends on AGP && X86
 	help
 	  This option gives you AGP support for the GLX component of XFree86 4.x
 	  on Intel 440LX/BX/GX, 815, 820, 830, 840, 845, 850, 860, 875,
